metadata.md

```diff
@@ -3,15 +3,25 @@ layout: default
 css_id: metadata
 ---
 
-# Metadata
+# Roles and Metadata
 
-Metadata files provide information that clients can use to make update decisions. Different metadata files provide different information. The various metadata files are signed by different keys as are indicated by the root role. The concept of roles allows TUF to only trust information that a role is trusted to provide.
+TUF uses metadata to create verifiable records about the state
+of a repository or application at a specified time and, as such, clients can use
+it to make decisions on which files to update. Different metadata files provide
+different information, and will be signed by different roles.
 
-The signed metadata files always include the time they were created and their expiration dates. This ensures that outdated metadata will be detected and that clients can refuse to accept metadata older than that which they've already seen.
+Roles define the set of actions a party can perform.The concept of roles allows
+TUF to only trust information provided by the correctly
+delegated party. The root role indicates which roles can sign for which projects.
```

Comment:
In TUF, we use "delegated" and "delegation" to refer to a _delegated role_.  A Targets role can transfer trust of packages to another role, and we sometimes refer to the deletator as a "delegated role."  I wonder if the use of "delegated party" might cause confusion.